### DISCUSSION TITLE
v0.8 is now available - how can we help?

### DISCUSSION THREAD

**Discussion by gerhard** [dagger/dagger#5584] [General]
We're continuously improving and adding features to Dagger. Occasionally, those improvements require us to make breaking changes to the Dagger API. Dagger 0.8 introduces several such breaking changes.

Our blog post explains the most important ones & helps you have a smooth upgrade process: https://dagger.io/blog/dagger-0-8

If you hit any issues with this new version, or need help upgrading, feel free to leave a comment below.

Happy Upgrading!

---

Thank you to everyone that contributed to Dagger v0.8! 🙌

Special thanks to @helderco 💪 for diligently working through all breaking changes. FWIW:
- https://github.com/dagger/dagger/discussions/5374
- [0.8.0 BREAKING CHANGES RELEASE - 🐙 GitHub Milestone](https://github.com/dagger/dagger/milestone/18?closed=1)

---

> [!IMPORTANT]  
> `v0.8.7` contains the latest fixes:
> - [🚙 Engine + 🚗 CLI `v0.8.7`](https://github.com/dagger/dagger/releases/tag/v0.8.7)
> - [🐹 Go SDK `v0.8.7`](https://github.com/dagger/dagger/releases/tag/sdk%2Fgo%2Fv0.8.7)
> - [🐍 Python SDK `v0.8.7`](https://github.com/dagger/dagger/releases/tag/sdk%2Fpython%2Fv0.8.7)
> - [⬢ Node.js SDK `v0.8.7`](https://github.com/dagger/dagger/releases/tag/sdk%2Fnodejs%2Fv0.8.7)
> - [🧪 Elixir SDK `v0.8.7`](https://github.com/dagger/dagger/releases/tag/sdk%2Felixir%2Fv0.8.7)

**Comment by gerhard** [comment]
We have discovered an issue with the latest `v0.8.2`:
- https://github.com/dagger/dagger/issues/5627

For now, stick with `v0.8.1`. Will update when we have released a new version with the fix.

**Reply by vito** [reply]
Fix is ready: https://github.com/dagger/dagger/pull/5628

**Comment by gerhard** [comment]
Mission accomplished, closing 🏁
